src/contexts/models/database/sequelize/sequelizeConnector.js

```diff
@@ -0,0 +1,64 @@
+
+const Sequelize = require('sequelize');
+module.exports = {
+    init,
+    saveContext,
+    getContexts,
+    deleteContext
+};
+
+let client;
+
+async function init(sequlizeClient) {
+    client = sequlizeClient;
+    await initSchemas();
+}
+
+async function initSchemas() {
+    const context = client.define('context', {
+        id: {
+            type: Sequelize.DataTypes.UUID,
+            primaryKey: true
+        },
+        name: {
+            type: Sequelize.DataTypes.STRING,
+            allowNull: false,
+            unique: true
+        }
+    });
+    await context.sync();
+}
+
+async function saveContext(id, contextName) {
+    const contextClient = client.model('context');
+
+    const duplicate = await contextClient.findOne({ where: { name: contextName } });
+    if (duplicate) {
+        throw Error('Duplicate context');
```

Comment:
need to throw here an error with 429 status code.
also need to verify that context_id doesn't exist.

Reply:
you defined `name` as `unique: true` in the sequelize data spec, so this `if` might be unnecessary and sequelize would throw an error in the `contextClient.create()` function, the error catching and 429 status code might be more relevant there